### DISCUSSION TITLE
Cannot preserve original text object dimensions after readjusting the fontsize

### DISCUSSION THREAD

**Discussion by emre-ozgun** [fabricjs/fabric.js#8367] [Q&A] [answered]
I've been grappling with this for over a week now. I can provide more information on request.

https://stackoverflow.com/questions/74065531/fabricjs-mapping-data-to-change-text-value-of-text-object-cannot-preserve-or

**Comment by ShaMan123** [comment, marked as answer]
https://github.com/fabricjs/fabric.js/pull/7981

**Reply by emre-ozgun** [reply]
Thank you so much!
